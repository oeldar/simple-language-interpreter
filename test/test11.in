a := 50;

if a > 0 then
	if a > 1000 then
		a := 1014;
print(a);

if a > 0 then
	if a < 100 then
		a := 512;
print(a);

if a > 1000 then
	if a > 0 then
		a := 2048;
print(a);
