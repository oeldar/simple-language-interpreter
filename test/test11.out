50
512
512
